libcontainerd/windowsoci/oci_windows.go

```diff
@@ -86,6 +86,8 @@ type Platform struct {
 	OS string `json:"os"`
 	// Arch is the architecture
 	Arch string `json:"arch"`
+	// OSVersion is the version of the operating system.
+	OSVersion string `json:"os.version,omitempty"`
```

Comment:
I was being consistent with `image.Image.OSVersion` and `manifestlist.PlatformSpec.OSVersion`. I had the same concern when I introduced those two fields, but `os.version` won the bikeshed contest.

If we don't think these need to be consistent, then we can certainly change it in a followup change.